### PR TITLE
Merge OfflineUseCases with Master

### PR DESCRIPTION
**Pull request by sehoven**
US 08.01.01 to US 08.04.01 have tests. ( This closes issue [#7](https://github.com/CMPUT301F16T01/Carrier/issues/7) )

### CODE REVIEW

**Comment by sehoven** [comment]
Kieter and Bennett pair programmed the tests for US 08.01.01 and US 08.02.01. Kieter will check over code for US 08.03.01 and US 08.04.01 tests.

File: app/src/main/java/comcmput301f16t01/github/carrier/SyncController.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    public boolean isOnline() {
+        return online;
+    }
+
+    public void setOnline(boolean status) {
+        online = status;
+    }
+
+    public Queue<Request> getRequestQueue() {
```
Comment (kieter) [inline app/src/main/java/comcmput301f16t01/github/carrier/SyncController.java:22]: Cool!

File: app/src/test/java/comcmput301f16t01/github/carrier/OfflineTests.java [hunk 2]
```diff
@@ -17,35 +19,110 @@ public void reset() {
...
+        // Setting up
+        Rider rider = new Rider("Mandy");
+        Request request = new Request(rider, new Location(), new Location());
+        RequestController rc = new RequestController();
+
+        // adding a request offline will send it to the queue in the SyncController
+        rc.addRequest(request);
+        // Tests that the request is in the SyncController queue waiting to be posted
+        //   this assumes that this is the only request in the queue
+        assertEquals("Request not in queue", request, sc.getRequestQueue().peek());
```
Comment (kieter) [inline app/src/test/java/comcmput301f16t01/github/carrier/OfflineTests.java:86]: So many 175 memories.

**Review by kieter — APPROVED** [review]
This is good. I like this.

**Pull request merged**